{{cookiecutter.github_repository}}/provisioner/roles/common/defaults/main.yml

```diff
@@ -31,4 +31,5 @@ base_ubuntu:
       - postgresql-client
       - libpq-dev
       - ca-certificates
+      - graphviz
```

Comment:
Can you put this in `project_data` role, as it is a very specific dependency of the django project.

Reply:
done